draft-ietf-quic-transport.md

```diff
@@ -4984,6 +4994,12 @@ Sequence Number:
 : The sequence number assigned to the connection ID by the sender.  See
   {{issue-cid}}.
 
+Retire Prior To:
+
+: A variable-length integer specifying the limit, for which all connection IDs
+  assigned to the receiver that have a sequence number less this value should be
+  retired.  See {{retiring-cids}}.  This field is only present for type=0x19.
```

Comment:
This doesn't say anything about what to do with the connection ID that is negotiated during the handshake, and the one attached to a server preferred address.